Support deploying by name with multiple deployments with the same name during `prefect deploy` (#10189)
PrefectHQ/prefect: docs/concepts/deployments-ux.md

```
</div>

> To deploy multiple deployments with the same name you can prefix the flow name in front of the deployment name:

Reviewer: ```suggestion
To deploy multiple deployments with the same name, you can prefix the deployment name with its flow name:
```
